README.md

```diff
@@ -1 +1,13 @@
-# flat-search
\ No newline at end of file
+# flat-search
+
+This is a small project for looki-up a flash according to the options stored in config file.
+
+This cli should be installed to run migrations:
+```
+https://github.com/mattes/migrate
+```
+Note: to run migrations please use the following command:
+```bash
+$GOPATH/bin/migrate -url mysql://[DB_USER_NAME]:[DB_USER_PASSWORD]@tcp\([IP_OR_HOSTNAME]:[PORT]\)/[DB_NAME] -path ./migrations/ up
+```
```

Comment:
You can use https://github.com/seesawlabs/mighero That package will allow you to read `DB_USER_NAME ` and other parameters from your config file.